resteasy-client/src/main/java/org/jboss/resteasy/client/jaxrs/internal/ClientInvocation.java

```diff
@@ -560,14 +560,18 @@ public ClientResponse invoke()
       }
       else
       {
-         return executorSubmit(client.asyncInvocationExecutor(), callback, extractor);
+         return executorSubmit(asyncInvocationExecutor(), callback, extractor);
       }
    }
 
+   public ExecutorService asyncInvocationExecutor() {
+       return client.asyncInvocationExecutor();
+   }
+

```

Comment:
This is just for convenience here and in ClientInvocationBuilder, isn't it?

Reply:
Well it's used, and this was the previous default. I assume this will be used by the non-MP resteasy client.

Reply:
So this is customised by the MP client.

Reply:
Thanks, just noticed (that's what happens why reviewing from github UI only and not checking out the code...)